aspnetcore/grpc/aspnetcore/sample/GrcpService/GreeterService2.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Greet;
+using Grpc.Core;
+
+namespace GrpcGreeter
+{
+    #region snippet
+    public class GreeterService : Greeter.GreeterBase
```

Comment:
Is it still `Greeter.GreeterBase`? I thought we changed the C# namespace to match your project now

Reply:
GreeterBase is a nested class of Greeter. The Greeter's namespace has been updated to the match the project.

Reply:
Whoops 🤦‍♂